index.html

```diff
@@ -890,7 +890,9 @@ <h3>Extensions to the <code>Performance</code> Interface</h3>
 following substeps:
 <ol style="list-style-type: lower-latin;">
 <li>Set <a>resource timing buffer full event pending flag</a> to true.</li>
-<li>Queue a task to run <a>fire a buffer full event</a>.</li>
+<li><a>Queue a task</a> on the <a href=
+  "https://w3c.github.io/performance-timeline/#dfn-performance-timeline-task-source">performance
+  timeline task source</a> test to run <a>fire a buffer full event</a>.</li>
```

Comment:
"test" here seems unrelated...

Reply:
Oops I was testing something... thanks for catching it